tgui/packages/tgui/interfaces/PortableChemMixer.js

```diff
@@ -0,0 +1,120 @@
+import { toTitleCase } from 'common/string';
+import { Fragment } from 'inferno';
+import { useBackend } from '../backend';
+import { AnimatedNumber, Box, Button, LabeledList, Section } from '../components';
+import { Window } from '../layouts';
+
+const byTitle = function (a, b) {
+  if (a.title > b.title) {
+    return 1;
+  }
+  if (a.title < b.title) {
+    return -1;
+  }
+  return 0;
+};
+
+export const PortableChemMixer = (props, context) => {
+  const { act, data } = useBackend(context);
+  const recording = !!data.recordingRecipe;
+  const beakerTransferAmounts = data.beakerTransferAmounts || [];
+  const beakerContents = recording
+    && Object.keys(data.recordingRecipe)
+      .map(id => ({
+        id,
+        name: toTitleCase(id.replace(/_/, ' ')),
+        volume: data.recordingRecipe[id],
+      }))
+    || data.beakerContents
+    || [];
+  data.chemicals.sort(byTitle);
```

Comment:
sortBy ftom common/collections

Reply:
Thank you for the advice, I have replaced the old sorting function with sortBy, it works great 👍